src/components/Table/index.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import StoryWrapper from '../StoryWrapper';
+import { composeTable, Props } from '.';
+import { TableRow } from '../TableRow';
+
+enum FilterType {
+	name = 'name',
+	dueAt = 'dueAt',
+	createdAt = 'createdAt',
+	completed = 'completed',
+}
+
+const Table1 = composeTable<FilterType>();
+export default {
+  title: 'Table',
+  component: Table1,
+} as Meta;
+
+const Template1: Story<Props<FilterType>> = (args) => (
+  <StoryWrapper>
+    <Table1 {...args} />
+  </StoryWrapper>
+);
+
+export const Default = Template1.bind({});
+const defaultColumns = [
+	{ dataName: FilterType.name, displayName: 'NAME' },
+	{ dataName: FilterType.dueAt, displayName: 'DUE' },
+	{ dataName: FilterType.createdAt, displayName: 'CREATED' },
+	{ dataName: FilterType.completed, displayName: 'COMPLETED' },
+	{ dataName: 'actions', displayName: 'actions', hidden: true },
+]
+
+Default.args = {
+	columns: defaultColumns,
+	searchTerm: 'Hello World',
+	searchColumn: FilterType.name,
+	onSearch: () => console.log('Sup'),
+	renderTableContent: (one = 'one', two = 2, three = 'III') => (
```

Comment:
Why are these default argument values? The component doesn't pass anything, and these could just be locally scoped to this function